main.js

```diff
@@ -38,6 +38,9 @@ const CLI_OPTIONS = {
   args: [],
   flags: [
     { name: 'config', type: 'string', default: 'config.json' },
+    // List of already processed (i.e. emailed) items. See EMPTY_STATE for content description.
+    { name: 'state', type: 'string', default: 'state.json' },
+    { name: 'no_sandbox', type: 'boolean' },
```

Comment:
Please update the docs in text.md to mention these.

Reply:
of course :-)